MAUI/Cartesian-Charts/DataLabels.md

```diff
@@ -146,3 +146,123 @@ chart.Series.Add(series);
 {% endtabs %}
 
 ![Applying series interior for data label in MAUI chart](DataLabel_images/maui_chart_data_label_with_series_brush.jpg)
+
+## Formatting Label Context
+
+The content of the label can be customized using the [LabelContext]() property. Following are the two options that are supported now,
+
+* [Percentage]() - This will show the percentage value of corresponding data point Y value
+
+* [YValue]() - This will show the corresponding Y value.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart IsTransposed="True">
+        . . .
+        <chart:ColumnSeries ItemsSource="{Binding Data}" 
+                            ShowDataLabels="True"
+                            XBindingPath="Name"
+                            YBindingPath="Height" 
+                            LabelContext="Percentage"/>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    chart.IsTransposed = true;
+    . . .
+    ColumnSeries series = new ColumnSeries()
+    {
+        ItemsSource = new ViewModel().Data,
+        XBindingPath = "Name",
+        YBindingPath = "Height",
+        ShowDataLabels = true,
+        LabelContext = LabelContext.Percentage
+    };
+
+    chart.Series.Add(series);
+    this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_data_label_context.png)
+
+## LabelTemplate
+
+The [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html) provides support to customize the appearance of the datalabel by using the [LabelTemplate]() property.
```

Comment:
The [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html) provides support to customize the appearance of the datalabel using the [LabelTemplate]() property.